jupyter/src/incremental_learning/job.py

```diff
@@ -272,6 +280,101 @@ def get_model_update_data(self) -> str:
     def get_model_blob(self) -> str:
         return self.model
 
+    def store(self, destination: Path) -> bool:
+        """
+        Store job with trained model into a file.
+
+        Args:
+            destination (Path): local path to where the job should be stored.
+
+        Returns:
+            bool: True if storing was successfull, False otherwise.
+        """
+        success = True
+        with gzip.open(destination, 'wt') as fp:
+            try:
+                json.dump(obj=self, fp=fp, cls=JobJSONEncoder)
+            except:
+                ex = sys.exc_info()
+                logger.error("Failed storing job {}: {}".format(self.name, ex))
+                success = False
+        return success
+
+    @classmethod
+    def asJob(cls, state: dict):
```

Comment:
nit: stick with snake case conventions used in this file
```suggestion
    def as_job(cls, state: dict):
```

Reply:
renamed